DOC-1661/ttp-step-4-restructuring-and-prod-migration (#614)
swan-io/docs: docs/topics/merchants/index.mdx

## Overview {#overview}

> Merchants are account holders who want to accept payments by card, SEPA Direct Debit, Internal Direct Debit, in-person cards, or French checks, either directly from a payment link or an invoice. 

Reviewer: ```suggestion
Merchants are account holders who want to accept payments by card (online and in person), SEPA Direct Debit, Internal Direct Debit, or French checks. They can do this either through a payment link or an invoice.
```

> Merchant payment methods are connected to a merchant profile, and a single merchant profile can have multiple payment methods.

Reviewer: ```suggestion
Merchant payment methods are connected to a merchant profile. A single merchant profile can have multiple payment methods.
```
